### DISCUSSION TITLE
Is this repository dead?

### DISCUSSION THREAD

**Discussion by riverar** [microsoft/Windows-Dev-Performance#84] [General]
Seems @bitcrazed @zooba and friends are busy with other projects and this repository isn't seeing much action. Issues aren't getting triaged, pull requests not getting reviewed, etc. Why should developers file their bugs here?

**Comment by zooba** [comment]
Responsiveness was always going to be the hardest part, but we're still definitely interested in hearing about the issues. We definitely have other projects that take our attention away from being public relations folk on GitHub, and only one of those is trying to get the feedback directed appropriately internally.

Basically all of these involve a large amount of bottom-up advocacy with the relevant teams to get the priorities raised - very few have been completely unknown, they just haven't risen to enough impact to annoy the teams (many developers have an annoying habit of just quietly working around things ;) ).

It's possible that in time it'll just get so overwhelming that we have to give up. That's a big part of the reason we've been angling for developer-specific feedback - because devs are underrepresented in other feedback channels, and that's who we've been wanting to amplify. But until that happens (and it's not actually my call - I'm a helper, not an owner here), having the feedback is more valuable than not having it, and we are trying to get it in front of the teams who can do something about it. Updating tags on GitHub is the step we're overlooking right now, sorry.

**Comment by bitcrazed** [comment]
What @Zooba said. 

Despite some of us moving into different roles/teams, some of us remain interested in this repo and respond as much as we're able. Other colleagues do now own & monitor this repo as best they can. Please be patient and do continue to share your issues and feedback - we'd rather receive the issues and eventually get around to triaging them than have the issue lost or discarded.

**Reply by riverar** [reply]
Thanks. I'm having a real hard time with this one.

It sounds like this repository is just an issues dump with no real plan. There is no official support here, no ROI for folks filing bugs, no feedback, no workflow, and no comms. ([I thought this was the case back in July 2020](https://twitter.com/WithinRafael/status/1288263448694943744), but was led to believe otherwise.) I fear this isn't what any of us wanted and will just further frustrate the dev community, continue to feed the narrative that Microsoft doesn't care about feedback, and ultimately just result in less engagement with devs.

**Reply by riverar** [reply]
I don't have any quick solutions to offer. But there has to be demonstrable ROI here and a steady flow of some sort of progress. Perhaps WinDev MVPs, like myself, can help repro/triage and Microsoft could start pumping items into their bug tracker? The tricky part is, of course, getting updates back out. Just my 2 seconds of brainstorming. I want to make this work!
